package.py

```diff
@@ -1,6 +1,6 @@
-name = ""
-title = ""
-version = "0.0.0"
+name = "ayon-addon-template"
+title = "Ayon-Addon-Template"
```

Comment:
For what it's worth, the `ayon-core` and `ayon-maya` repositories have things like:
```
name = "core"
title = "Core"
```
And
```
name = "maya"
title = "Maya"
```

Should this then be:
```suggestion
name = "addon-template"
title = "Addon-Template"
```
To match along those lines? Or would that just add to confusion regarding this template?

Reply:
That's a fair point if it's already established like that.